src/client/pages/Offer/Introduction/DetailsModal/Details/components/DetailInput.tsx

```diff
@@ -47,7 +46,7 @@ export const DetailInput: React.FC<DetailInputProps &
       showErrorMessage={false}
       errors={getIn(formikProps.errors, formikName)}
       touched={getIn(formikProps.touched, formikName)}
-      autoComplete="off"
+      {...formikProps.getFieldProps(formikName)}
```

Comment:
Does this mean we pass "autoComplete" to formik instead?

Reply:
I have removed autoComplete to simplify code as I couldn't see it bring any value. 
Furthermore Chrome ignores the autoComplete='off' from version 34. https://stackoverflow.com/questions/31612867/autocomplete-off-not-working-in-chrome

I am not sure if Formik handle autoComplete.

Reply:
Hmm interesting. I guess it's not a standard. I'm okey not using it.